index.html

```diff
@@ -2913,6 +2913,12 @@ <h2 id="sec-1">Why Are Programmers So Intense About Languages?</h2>
             </sup>
             <p>Actually, programmers don&rsquo;t always talk as much as chat online, often using Slack. Slack is an Internet startup that provides a chat service organized around the needs of geographically diverse groups. A large part of its appeal is its simplicity and the ease with which it integrates with other products. Chat channels can quickly come to represent all the work that&rsquo;s going on inside a company, as a set of streams of text, which can then be searched. Slack provides a short-term institutional memory, which has made a lot of nerds happy. It&rsquo;s growing quickly. Slack&rsquo;s investors recently valued it at $2.6 billion; by the time this issue is published, it will probably be worth more than planet earth.</p>
          </div>
+         <div class="footdef">
+            <sup>
+                <a id="fn.40" name="fn.40" class="footnum" href="#fnr.40">40</a>
+            </sup>
+            <p>A wristband with a built in RF and NFC communicator that acts as your room key, park ticket, and personal identifier. <a href="http://www.wired.com/2015/03/disney-magicband/">Wired did a great piece</a> on the development on the development of the project</p>
```

Comment:
Two more quick suggestions. 

RFID instead of RF.

built-in instead of built in.